packages/mobile/src/types/declarations.d.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="@testing-library/jest-native" />
```

Comment:
is this just a comment or is it functional?

Reply:
ah good callout, yes this is to enable jest expect extension with typescript, allows for `expect().toBeOnTheScreen()` to typecheck

Reply:
cool `///` looked weird to me

Reply:
yeah its some declarations.d.ts way to add a type import: https://github.com/testing-library/jest-native#typescript-support